### DISCUSSION TITLE
Why replace react-virtualized with react-virtuoso?

### DISCUSSION THREAD

**Discussion by tianzhich** [nosferatu500/react-sortable-tree#33] [General]
Hi @nosferatu500, the original code use react-virtualized for the virtulization. But you replace it with react-virtuoso. Are there any considerations? Or why didn't you replace it with the newer react-window? 

Thank you.

**Comment by nosferatu500** [comment]
Hi!

First, I made this fork only for myself and did not plan to share it with others. But seeing that many people had problems with the original version at that time, I decided to share my fork.

Secondly, I tested `react-virtualized`, `react-window` and `react-virtuoso` on my very specific dataset and `react-virtuoso` showed the best performance at that time.

btw, I no longer use `react-sortable-tree` in my project and have completely switched to [react-dnd-treeview](https://github.com/minop1205/react-dnd-treeview). I keep reviewing PRs and updating deps from time to time, but I don't have time to refactor/fix any bugs in [react-sortable-tree](https://github.com/nosferatu500/react-sortable-tree).

**Reply by tianzhich** [reply]
Thanks mate for your quick reply. I check the `react-dnd-treeview` but it not meet my requirements (it seems not provide the virtualization and scrollIntoSearch etc...). So I decide to use react-sortable-tree, based on your fork.

I introduced the code in my project. If there are some bugs i encouterd i'm glad to sync these changes to this fork.

btw, thanks for your recommendations about the tree component. And the most important, thanks for your contribution to this awesome library.

**Reply by tianzhich** [reply]
BTW, how you tested these 3 packages? If you tested in codesandbox could you share with me? I also want to do some performance testing.

**Reply by nosferatu500** [reply]
Sorry, I did this over a year ago on my work Mac and don't have access to it now.
